### PR TITLE
Fix/srv subsrv logs

### PR DESCRIPTION
**Pull request by AlvaroVega**
related https://github.com/telefonicaid/iotagent-manager/issues/211

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,5 @@
+- Fix: set service subservice in logs of redirector and configurations
+- Fix: set new trans and corr or use it if provided in logs
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:2]: #211 should be mentioned in one of these?

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:2]: Not sure about 211 will be fixed with this PR

Comment (fgalan) [reply CHANGES_NEXT_RELEASE:2]: Ok, I missinterpret the "related #211" in this PR 👍 

NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**